### PR TITLE
removed old lg workflow, updated new workflow airflow payload

### LINKED ISSUE #1775: remove old performance test workflow (non airflow)
Current looking glass performance testing tool is getting archived. 
Need to remove the old workflow and update payload for the airflow workflow to not pass in a specific branch.

### PR DESCRIPTION
**Pull request by petejanuszewski1**
Closes #1775

### CODE REVIEW

File: .github/workflows/woodwork_main_airflow_performance_tests.yaml [hunk 1]
```diff
@@ -34,11 +34,9 @@ jobs:
           --data-raw '{
             "conf": {
                   "description": null,
-                  "lg_branch": "lg_baton_airflow",
```
Comment (ParthivNaresh) [inline .github/workflows/woodwork_main_airflow_performance_tests.yaml:37]: no need to specify the branch?

Comment (ParthivNaresh) [reply .github/workflows/woodwork_main_airflow_performance_tests.yaml:37]: probably defaults to `main`?

**Review by ParthivNaresh — APPROVED** [review]
LGTM

File: .github/workflows/woodwork_main_airflow_performance_tests.yaml (hunk 1 above)
Comment (petejanuszewski1) [reply .github/workflows/woodwork_main_airflow_performance_tests.yaml:37]: exactly, default is main and that is set in Airflow. That branch was added while we were still in the development phase.

**Pull request merged**